From: Helmut Grohne <helmut@subdivi.de>
Date: Sun, 4 Nov 2018 21:35:11 +0800
Subject: stop using libtool-bin

---
 Makefile.am | 4 ++--
 tests/setup | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 13f74c9..df020b1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -118,8 +118,8 @@ endif
 
 ### Tests (make check)
 
-TESTS = tests/optionaltest tests/mantest tests/jqtest tests/onigtest tests/shtest tests/utf8test tests/base64test
-TESTS_ENVIRONMENT = NO_VALGRIND=$(NO_VALGRIND)
+TESTS = tests/mantest tests/jqtest tests/onigtest tests/shtest
+TESTS_ENVIRONMENT = NO_VALGRIND=$(NO_VALGRIND) LIBTOOL="$(LIBTOOL)"
 
 
 ### Building the manpage
diff --git a/tests/setup b/tests/setup
index e000637..984cea7 100755
--- a/tests/setup
+++ b/tests/setup
@@ -13,7 +13,7 @@ JQBASEDIR=$JQTESTDIR/..
 JQ=$JQBASEDIR/jq
 
 if [ -z "${NO_VALGRIND-}" ] && which valgrind > /dev/null; then
-    VALGRIND="libtool --mode=execute -- valgrind \
+    VALGRIND="$LIBTOOL --mode=execute -- valgrind \
     		      	       --error-exitcode=1 --leak-check=full \
 		      	       --suppressions=$JQTESTDIR/onig.supp"
     Q=-q
