From: =?utf-8?b?IkNoYW5nWmh1byBDaGVuICjpmbPmmIzlgKwpIg==?=
 <czchen@debian.org>
Date: Sun, 4 Nov 2018 21:34:10 +0800
Subject: Do not distribute COPYING, fixes lintian warning

Forwarded: not-needed
Last-Update: 2015-10-11
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 1df4143..7387506 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -238,7 +238,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS=--with-oniguruma=builtin
 
 # README.md is expected in GitHub projects, good stuff in it, so we'll
 # distribute it and install it with the package in the doc directory.
-dist_doc_DATA = README.md NEWS.md COPYING AUTHORS
+dist_doc_DATA = README.md NEWS.md AUTHORS
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libjq.pc
