From: =?utf-8?b?IkNoYW5nWmh1byBDaGVuICjpmbPmmIzlgKwpIg==?=
 <czchen@debian.org>
Date: Sun, 4 Nov 2018 21:34:10 +0800
Subject: patch Makefile to always generate the manpage.

Forwarded: not-needed
Last-Update: 2014-06-21
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index c3a771f..6f44e5a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -106,6 +106,7 @@ TESTS_ENVIRONMENT = NO_VALGRIND=$(NO_VALGRIND)
 
 man_MANS = jq.1
 .PHONY: real_docs
+ENABLE_DOCS=1
 if ENABLE_DOCS
 jq.1: $(srcdir)/docs/content/3.manual/manual.yml
 	$(AM_V_GEN) ( cd ${abs_srcdir}/docs; '$(BUNDLER)' exec rake manpage ) > $@ || { rm -f $@; false; }
